.github/workflows/ci-actions.yml

```diff
@@ -344,13 +343,14 @@ jobs:
         run: tar -xzf maven-repo.tgz -C ~
       - name: Build with Gradle
         uses: eskatos/gradle-command-action@v1
+        timeout-minutes: 60
         env:
           GRADLE_OPTS: -Xmx1408m
         with:
           gradle-version: wrapper
           wrapper-directory: integration-tests/gradle
           build-root-directory: integration-tests/gradle
-          arguments: clean test -i -S --stacktrace --no-daemon
+          arguments: clean test --debug -S --stacktrace --no-daemon
```

Comment:
Looks like the debug flag is still there :).

Reply:
yes, corrected :)